reformat HGVS records

sarahhunt opened the pull request.

jeromekelleher commented on src/main/resources/avro/alleleAnnotations.avdl:
```diff
@@ -107,6 +107,22 @@ record VariantAnnotationSet {
 /**
+A HGVSAnnotation record holds Human Genome Variation Society descriptions
+of the variant allele with respect to a transcript.
+*/
```
A stupid question: are there any requirements about which of these fields can be null at the same time? I.e., can we have genomic, coding, or protein set, in any combination?

jeromekelleher commented on src/main/resources/avro/alleleAnnotations.avdl:
```diff
@@ -107,6 +107,22 @@ record VariantAnnotationSet {
+*/
+record HGVSAnnotation {
+
+  /** HGVS formatted annotation at genomic level  */
```
I'm afraid I don't find these comments hugely enlightening. I already know that it's a "HGVS formatted annotation" since it's held in this struct. Would it be possible to get a slightly higher-level description? Perhaps some links to the HGVS details?

sarahhunt commented:
@jeromekelleher - I see what you mean - I've removed the redundant comments. It would be cumbersome to describe HGVS in great detail here, so I've added a link to the recomendation. What do you reckon?

jeromekelleher commented:
Looks good to me @sarahhunt, +1

diekhans commented:
+1 on the hgvs change.  Linking to canonical external
documentation, when it exists, is far better that writing
a condensed `reader's digest' version

Sarah Hunt <notifications@github.com> writes:
> @jeromekelleher - I see what you mean - I've removed the redundant comments. It
> would be cumbersome to describe HGVS in great detail here, so I've added a link
> to the recomendation. What do you reckon?
> 
> —
> Reply to this email directly or view it on GitHub.*
>
